Ginger/Ginger/GeneralWindows/Run Set Pages/NewRunSetPage.xaml.cs

```diff
@@ -1252,52 +1262,9 @@ private void UpdateRunnersTabHeader()
             }
         }
         
-        private void CreateRunSetShortCut(string RunSet, string Env)
-        {
-            object shDesktop = (object)"Desktop";
-            WshShell shell = new WshShell();
-            string shortcutAddress = (string)shell.SpecialFolders.Item(ref shDesktop) + @"\Ginger " + RunSet + " " + Env + ".lnk";
-            IWshShortcut shortcut = (IWshShortcut)shell.CreateShortcut(shortcutAddress);
-            shortcut.Description = "Ginger Solution=" +  WorkSpace.Instance.Solution.Name + ", RunSet=" + RunSet + ", Env=" + Env;
-            string GingerPath = System.Reflection.Assembly.GetExecutingAssembly().Location;
-            string SolFolder =  WorkSpace.Instance.Solution.Folder;
-            if (SolFolder.EndsWith(@"\"))
-            {
-                SolFolder = SolFolder.Substring(0, SolFolder.Length - 1);
-            }
-            shortcut.TargetPath = GingerPath; 
-            string sConfig = "Solution=" + SolFolder + Environment.NewLine;
-            sConfig += "Env=" + Env + Environment.NewLine;
-            sConfig += "RunSet=" + RunSet + Environment.NewLine;
-            string sConfigFile = SolFolder + @"\Documents\RunSetShortCuts\" + RunSet + "_" + Env + ".Ginger.Config";
-
-            if (!System.IO.Directory.Exists(SolFolder + @"\Documents\RunSetShortCuts\")) { System.IO.Directory.CreateDirectory(SolFolder + @"\Documents\RunSetShortCuts\"); }
-            System.IO.File.WriteAllText(sConfigFile, sConfig);
-
-            shortcut.Arguments = "ConfigFile=\"" + sConfigFile + "\"";
-            shortcut.Save();
-            Reporter.ToUser(eUserMsgKey.ShortcutCreated, shortcut.Description);
-        }
-
-        private void createShortcutRunset_Click(object sender, RoutedEventArgs e)
-        {
-            AutoLogProxy.UserOperationStart("CreateShortcutButton_Click");
-
-            char[] invalidChars = System.IO.Path.GetInvalidFileNameChars();
-
-            if (mRunSetConfig.Name.IndexOfAny(invalidChars) >= 0 )
-            {
-                foreach (char value in invalidChars)
-                {
-                    if(mRunSetConfig.Name.Contains(value))
-                        mRunSetConfig.Name = mRunSetConfig.Name.Replace(value, '_');
-                }
-            }
-
-            CreateRunSetShortCut(mRunSetConfig.Name, xRunsetEnvironmentCombo.Text);
+        
 
```

Comment:
are we doing Run set name invalid chars check in new wizard?
foreach (char value in invalidChars)
                {
                    if(mRunSetConfig.Name.Contains(value))
                        mRunSetConfig.Name = mRunSetConfig.Name.Replace(value, '_');
                }